### PR TITLE
Ent - IngestSLSAID

### PR DESCRIPTION
**Pull request by desmax74**
# Description of the PR

Ent backend

- `IngestSLSAID` (with bulk) implementations
-  `IngestSLSA`, removed
-   changes consistently all of the tests involved

Refers to https://github.com/guacsec/guac/issues/1198

# PR Checklist

- [x] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [x] All new changes are covered by tests
- [ ] If GraphQL schema is changed, `make generate` has been run
- [ ] If `collectsub` protobuf has been changed, `make proto` has been run
- [ ] All CI checks are passing (tests and formatting)
- [ ] All dependent PRs have already been merged

### CODE REVIEW

File: pkg/assembler/backends/ent/backend/slsa.go [hunk 1]
```diff
@@ -140,15 +142,18 @@ func upsertSLSA(ctx context.Context, client *ent.Tx, subject model.ArtifactInput
 		UpdateNewValues().
```
Comment (mrizzi) [inline pkg/assembler/backends/ent/backend/slsa.go:142]: This has to be replaced with `DoNothing()`
```suggestion
		DoNothing().
```

**Review by mrizzi — CHANGES_REQUESTED** [review]
`DoNothing()` has to be applied.

File: pkg/assembler/backends/ent/backend/slsa.go (hunk 1 above)
Comment (desmax74) [reply pkg/assembler/backends/ent/backend/slsa.go]: It skip the update and the test fail with this change

Comment (desmax74) [reply pkg/assembler/backends/ent/backend/slsa.go]: ok

**Comment by desmax74** [comment]
@mrizzi

**Comment by pxp928** [comment]
We can now make these updates on main as needed. :)

**Pull request closed without merging**